parsec/core/fs/workspacefs/workspacefs.py

```diff
@@ -4,16 +4,32 @@
 from collections import defaultdict
 from contextlib import asynccontextmanager
 from pathlib import Path
-from typing import TYPE_CHECKING, AsyncIterator, Awaitable, Callable, Dict, List, Tuple, cast
+from typing import TYPE_CHECKING, AsyncIterator, Awaitable, Callable, cast
 
 import attr
 import structlog
 import trio
 
-from parsec._parsec import CoreEvent, DateTime, FileManifest, RealmStatusRepOk, Regex
-from parsec.api.data import AnyRemoteManifest, BlockAccess
-from parsec.api.data import FileManifest as RemoteFileManifest
-from parsec.api.protocol import MaintenanceType, RealmID, UserID
+from parsec._parsec import (
+    BlockAccess,
+    CoreEvent,
+    DateTime,
+    EntryID,
+    EntryName,
+    FileManifest,
```

Comment:
`FileManifest` was previously imported from `parsec.api.data` as `RemoteFileManifest`, and from `parsec._parsec` as `FileManifest`. What's the difference between these two? Is the alias useless? Do we have two different classes with the same name?

Reply:
`FileManifest` from `parsec.api.data` is a re-export of `parsec._parsec`